moveit_ros/planning_interface/move_group_interface/src/wrap_python_move_group.cpp

```diff
@@ -504,8 +507,26 @@ class MoveGroupInterfaceWrapper : protected py_bindings_tools::ROScppInitializer
       traj_obj.setRobotTrajectoryMsg(ref_state_obj, traj_msg);
 
       // Do the actual retiming
-      trajectory_processing::IterativeParabolicTimeParameterization time_param;
-      time_param.computeTimeStamps(traj_obj, velocity_scaling_factor);
+      if (algorithm == "" || algorithm == "iterative_time_parameterization")
```

Comment:
I would prefer an empty argument to raise an error.

Reply:
Updated.